schema/gks-core/gks-core-source.yaml

```diff
@@ -134,14 +131,9 @@ $defs:
       name:
         type: string
         description: A primary name for the concept.
-      primaryCode:
-        $ref: "#/$defs/code"
-        description: >-
-          A primary code for the concept that is used to identify the concept in a terminology or code system.
-          If there is a public code system for the primaryCode then it should also be specified in the mappings
-          array with a relation of 'exactMatch'. This attribute is provided to both allow a more technical code
-          to be used when a public Coding with a system is not available as well as when it is available but
-          should be identified as the primary code.
+      primaryCoding:
+        $ref: "#/$defs/Coding"
+        description: A primary coding for the concept.
```

Comment:
I feel like we could give some additional guidance about how to populate this attribute - either here or in the web docs.  

In our discussions, I think we decided that this is used to hold _"a code that is considered the primary representation of the concept, as defined or used in the data provider's system"_.  Adding language like this could help implementers.